### DISCUSSION TITLE
Auth token configured in settings.xml, mvn clean site gives warning of unauthorized error.

### DISCUSSION THREAD

**Discussion by anantk24** [microsoft/gctoolkit#29] [Q&A] [answered]
I have configured username/token maven server configurations. 
Token has access of following scopes
- repo Full control of private repositories
    -  repo:status Access commit status
    -  repo_deployment Access deployment status
    -  public_repo Access public repositories
    -  repo:invite Access repository invitations
    -  security_events Read and write security events

-  workflow Update GitHub Action workflows
-  write:packages Upload packages to GitHub Package Registry 
   -  read:packages Download packages from GitHub Package Registry
- delete:packages Delete packages from GitHub Package Registry
-  admin:org Full control of orgs and teams, read and write org projects
   -  write:org Read and write org and team membership, read and write org projects
   -  read:org
   
Warning :
 Original error: Could not transfer metadata com.microsoft.gctoolkit:parser:2.0.1-SNAPSHOT/maven-metadata.xml from/to github (https://maven.pkg.github.com/microsoft/*): Not authorize
d , ReasonPhrase:Unauthorized.

Anything am I missing?

Thanks.

**Comment by karianna** [comment]
I assume your GH PAT token is in your ~/.m2/settings.xml file?

If so, next question is, do you belong to an organization that has SSO sign-in for GitHub?

@dsgrieve Anything else we should check?

**Reply by dsgrieve** [reply]
-X output might provide some additional clues.

**Reply by anantk24** [reply]
Nope . We don't use SSO sign-in for github. I got the error, Mistakenly space character was added in while pasting token into settings.xml . Corrected it, working now. Thanks for the help 👍 .

**Comment by anantk24** [comment, marked as answer]
Solved issue, by correcting token value.
